scout/parse/case.py

```diff
@@ -418,6 +419,11 @@ def parse_individual(sample):
 
     ind_info["tissue_type"] = sample.get("tissue_type")
 
+    ind_info["chromograph_images"] = sample.get(
+        "chromograph_images",
+        {"upd_regions": "upd", "upd_sites": "upd_chr", "coverage": "", "autozygous": ""},
```

Comment:
Here you give a default, but e.g. in the loop to add individuals in the case.html insert_multipl_image_panel() loop you check if it is true. Perhaps just give it a false / none value if not given?

Reply:
Thanks. Removed defaults.